### DISCUSSION TITLE
Another Billing Provider

### DISCUSSION THREAD

**Discussion by paulmercs** [maartenpaauw/filament-cashier-billing-provider#11] [General]
Can you  add another billing provider like Xendivel? because stripe is not available here in my country. Thank you!

**Comment by maartenpaauw** [comment]
This billing provider is designed specifically for Laravel Cashier. Since Laravel does not offer an Xendivel Cashier package, I do not plan to incorporate support for Xendivel.

Feel free to write a custom billing provider: https://filamentphp.com/docs/3.x/panels/tenancy#writing-a-custom-billing-integration
